canvasapi/account.py

```diff
@@ -471,19 +479,22 @@ def update_role(self, role_id, **kwargs):
         )
         return Role(self._requester, response.json())
 
-    def get_enrollment(self, enrollment_id, **kwargs):
+    def get_enrollment(self, enrollment, **kwargs):
         """
         Get an enrollment object by ID.
 
         :calls: `GET /api/v1/accounts/:account_id/enrollments/:id \
         <https://canvas.instructure.com/doc/api/enrollments.html#method.enrollments_api.show>`_
 
-        :param enrollment_id: The ID of the enrollment to retrieve.
-        :type enrollment_id: int
+        :param enrollment_id: The Object or ID of the enrollment to retrieve.
```

Comment:
Like above, `enrollment` vs `enrollment_id` in the docstring since it's been updated. This probably exists elsewhere -- double check.